app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
```

Comment:
@eshadatta I actually think I might have added that case at one point but I don't see specs and I don't see any reason so I think it's save to drop.